### ISSUE TITLE
WAILA/Entity Names Feature Request

### ISSUE THREAD

**Issue by scribe-of-days** [ercanserteli/xercamods#56]
Hey Xerca,

Great job with all the updates man, thank you for keeping up the hard work! I've been playing in a modpack recently that includes "WTHIT Forge Edition" which is essentially a port for WAILA/WAWLA, and indicates the name of a block/item/object that you're looking at, at the top of the screen. Please see the attached snapshot:

![image](https://user-images.githubusercontent.com/81274800/165434385-2a92a019-8c89-4d78-8236-f83e14d1a55a.png)

Basically, every painting shows up as "entity.xercapaint.canvas" when you look at it, and as a feature request, it'd be really nice if the name of the signed painting would show up (as well as who signed it). I hope this is something you might consider for future releases!

Thanks again :)

**Comment by ercanserteli** [comment]
Sadly it wasn't possible to show custom info here, but it shows a correct name for the canvas and other entities now
